### PR TITLE
add tests for improper-usage-messages

### PR DESCRIPTION
**Pull request by abouthiroppy**
refs: https://github.com/avajs/ava/pull/1457

### CODE REVIEW

**Comment by abouthiroppy** [comment]
@novemberborn PTAL

File: test/reporters/improper-usage-messages.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
```
Comment (kevva) [inline test/reporters/improper-usage-messages.js]: Remove this blank line :).

File: test/reporters/improper-usage-messages.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+const test = require('tap').test;
+const chalk = require('chalk');
```
Comment (kevva) [inline test/reporters/improper-usage-messages.js]: `chalk` isn't used anywhere.

**Comment by abouthiroppy** [comment]
@kevva fixed;)

**Comment by novemberborn** [comment]
Thanks @abouthiroppy!

**Pull request merged**